docs/faq.rst

```diff
@@ -12,6 +12,39 @@ Yes, the `deduplication`_ technique used by
 |project_name| makes sure only the modified parts of the file are stored.
 Also, we have optional simple sparse file support for extract.
 
+If you use tools like Borg (or rsync, tar, cp and most other backup tools) to back up
```

Comment:
`like Borg (and most others)` -- since backup tools that do their own snapshotting one way or another tend to have this as a prominent feature?